edb/schema/delta.py

```diff
@@ -1405,6 +1406,36 @@ def _cmd_tree_from_ast(cls, schema, astnode, context):
                 schema,
                 context.modaliases,
             )
+        elif field.name == 'required' and new_value is False:
```

Comment:
`new_value is False` is a bit weird -- can it be replaced with `not new_value`?

Reply:
Add a comment if `None` is a legitimate value you're guarding against.

Reply:
Hmmm, the value comes from the AST, so it should never be `None`, unless the AST is malformed. AST creation is entirely under our control, so a malformed AST would be a separate bug which we don't need to handle here. I'll change the expression to `not new_value`.